packages/mobile/src/app/App.tsx

```diff
@@ -45,71 +46,30 @@ BigNumber.config({
   },
 })
 
-interface State {
-  reactInitTime?: number
-  reactLoadTime?: number
-}
-
 // Enables LayoutAnimation on Android. It makes transitions between states smoother.
 // https://reactnative.dev/docs/layoutanimation
 // Disabling it for now as it seems to cause blank white screens on certain android devices
 // if (Platform.OS === 'android' && UIManager.setLayoutAnimationEnabledExperimental) {
 //   UIManager.setLayoutAnimationEnabledExperimental(true)
 // }
 
-const createEventListener: (
-  eventName: string,
-  callback: (reactInitTime: string) => void
-) => EmitterSubscription | null = (
-  eventName: string,
-  callback: (reactInitTime: string) => void
-): EmitterSubscription | null => {
+function getEventEmitter() {
   if (Platform.OS === 'android') {
-    return new NativeEventEmitter().addListener(eventName, callback)
+    return new NativeEventEmitter()
```

Comment:
nit: More of a stylistic thing but may make sense to replace the `if...else` with a ternary to keep it clean

Reply:
I found a Platform.select function that seems to be pretty useful and makes it look cleaner.